pkg/sdk/informer.go

```diff
@@ -56,7 +56,7 @@ func NewInformer(resourcePluralName, namespace string, resourceClient dynamic.Re
 		numWorkers:         n,
 	}
 
-	resyncDuration := time.Duration(resyncPeriod) * time.Second
+	resyncDuration := resyncPeriod * time.Second
```

Comment:
No need to multiply by time.Second here.
The user is allowed to pass in any Duration to sdk.Watch().